### PR TITLE
Config with default template 2023-01-27

### PR DESCRIPTION
**Pull request by gforcada**
Configured with https://github.com/plone/meta/pull/27

### CODE REVIEW

**Comment by gforcada** [comment]
@jenkins-plone-org please run jobs

File: plone/schemaeditor/browser/schema/add_field.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def create(self, data):
 
         # remove fieldset_id from data
         if "fieldset_id" in data:
-            fieldset_id = data.pop("fieldset_id")
+            _ = data.pop("fieldset_id")
```
Comment (mauritsvanrees) [inline plone/schemaeditor/browser/schema/add_field.py:50]: The whole `if` statement could be replaced by `data.pop("fieldset_id", None)`. But we don't have to do it in this PR. Keeping the manual changes small is good.

**Review by mauritsvanrees — APPROVED** [review]
LGTM.
One minor inline comment, but can be ignored.
Can be merged when green.

**Comment by mauritsvanrees** [comment]
One unrelated robot failure. I merge. Thanks!

**Pull request merged**